src/lavinmq/unacked_message.cr

```diff
@@ -0,0 +1,18 @@
+module LavinMQ
+  struct UnackedMessage
+    include SortableJSON
+    getter message_tag, consumer_tag, channel, delivered_at
+
+    def initialize(@channel : LavinMQ::Client::Channel, @message_tag : UInt64, @delivered_at : Time::Span, @consumer_tag : String? = nil)
+    end
+
+    def details_tuple
+      {
+        message_tag:         @message_tag,
```

Comment:
message_tag should be delivery_tag no? I dont think we use the name message_tag anywhere else.